### ISSUE TITLE
Add support for more than 1 custom light show on the USB drive

### ISSUE THREAD

**Issue by jsevedge** [teslamotors/light-show#48]
I was able to build a custom light show in a few hours using the directions provided in this repo, thanks much!

**Feature Request**: It would be great if the Tesla car logic for light show discovery was enhanced to support multiple custom light shows, either using a config driven format such as a JSON/YAML config file with some metadata about the location of each light show on the usb drive OR even simpler each top level folder (if not named LightShow) could contain a custom light show and the name of the folder would show up in a picker on the screen.

**Comment by MrPaulAR** [comment]
Duplicate of https://github.com/teslamotors/light-show/issues/8

**Comment by jLynx** [comment]
Feels like this feature is on its way looking at the new UI 
![20221226_090217](https://user-images.githubusercontent.com/4393979/209480642-aa37ff7b-6318-433d-8134-96a16e5034ed.jpg)
 Any update?

**Comment by mknippen** [comment]
+1 this is like a must-have feature now with the new UI!

If we could simply just name all of the files differently inside the LightShow folder, I think that would be more than enough.
On the system side, just search for all files that are .fseq inside the folder, and check to make sure a matching song (same name) is present when you tap it.

**Comment by simon2022** [comment]
should be work now with the 2023 holiday update

**Comment by jLynx** [comment]
@simon2022 are you saying the holiday uodate adds support for multiple light show files? Are you able to please post a photo of the screen showing this?

I have not received the update yet, so I cannot confirm

**Comment by Woutch** [comment]
Yes, it was mantioned in the details. However it is not clear how to get this working...
It says all show must be put in the LightShow folder, but how to do this if it needs to be named the same? Tried subfolders or rename in the lightshow folder, but does not seem to work...

![image](https://github.com/teslamotors/light-show/assets/3136903/62fcb616-6b05-4a63-ab31-c193aa2e6799)

**Comment by jLynx** [comment]
Oh I totally missed that! Awesome, try put them in folders like show1, show2 etc. I will have to wait till I get the update for my car

**Comment by jLynx** [comment]
Ok I just got the update and confirm this now works. In order for it to work your shows need to be in the /LightShow folder and both the .fseq and .wav/.mp3 file need to have the same same. This is what mine looks like:
![image](https://github.com/teslamotors/light-show/assets/4393979/d6d284d3-e32e-4433-a255-22c3002973a3)

And then in the car you will see them all show up, such as this:
![image](https://github.com/teslamotors/light-show/assets/4393979/d3cf714a-ec6e-417d-b4a3-5ed1908b04ab)

**Comment by denis-rafailov** [comment]
I guess then that ticket can be closed?
